examples/tiny_skia_render/src/main.rs

```diff
@@ -55,9 +56,7 @@ fn main() {
     builder.push_default(&brush_style);
 
     // Set default font family
-    let font_stack = FontStack::Source(Cow::Borrowed("system-ui"));
-    let font_stack_style = StyleProperty::FontStack(font_stack);
-    builder.push_default(&font_stack_style);
+    builder.push_default(&GenericFamily::SystemUi.into());
```

Comment:
Could make `push_default` take `impl Into<StyleProperty>` and eliminate this `.into()`.

Reply:
I like that, I'll add it.